terracumber_config/tf_files/SUSEManager-build-validation-continuous-NUE.tf

```diff
@@ -0,0 +1,1276 @@
+// Mandatory variables for terracumber
```

Comment:
Let's rename this as continuous build validation please. So, it really breaks the regex as a different kind of entity.